src/components/DataTable/index.jsx

```diff
@@ -54,7 +54,7 @@ class DataTable extends Component {
   };
 
   handleHeaderClick = ({ target }) => {
-    if (this.props.onHeaderClick) {
+    if (this.props.onHeaderClick && target.id !== 'Tags') {
```

Comment:
`DataTable` should not have knowldege of `Tags`. Put this logic in `TasksTable`.